tensor_comprehensions/pybinds/pybind_engine.cc

```diff
@@ -59,13 +60,17 @@ PYBIND11_MODULE(tc, m) {
     std::cerr << "\n PyTorch installation is missing, binary will be useless \n"
               << e.what() << std::endl;
   }
-  py::class_<tc::ATenCompilationUnit>(m, "ATenCompilationUnit")
+  py::class_<tc::ATenCompilationUnit<tc::CudaTcExecutor>>(
```

Comment:
@prigoyal  do your python tests still pass?

Reply:
@ftynse  @prigoyal  ```test.sh``` pass but if needed it would be good to add a bit of python tests to it?

Reply:
yeah, that would force me to finally install with pytorch :)

Reply:
test.sh doesn't run python tests.